model/Core/Classes/Build.md

```diff
@@ -0,0 +1,74 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# Build
+
+## Summary
+
+Class that describes a build of software/artifacts.
+
+## Description
+
+A build is a representation of the process in which a piece of software or
+artifact is built. It encapsulates information related to a build process and
+provides an element from which relationships can be created to relate the build
+to the inputs/outputs, and entities related to the build (e.g. builders,
+identities, etc.).
+
+Definitions of "BuildType", "ConfigSource", "Parameters" and "Environment" follow
+those defined in [SLSA provenance](https://slsa.dev/provenance/v0.2).
+
+<!--
+The build will include relationships such as:
+
+<Build> BUILT_BY <SPDX_Element> 
+<Build> BUILD_INITIATED_BY <Subject/Actor>
+<Package> BUILD_INPUT_OF <Build>
+<Package> BUILD_OUTPUT_OF <Build>
+
+For BUILT_BY:
+At the moment it can point to an artifact to signify a builder, but can point
+to a "Provider/Service" element in the future which can encapsulate more
+details of the builder if needed.
+-->
+
+## Metadata
+
+- name: Build
+- SubclassOf: Element
+- Instantiability: Concrete
+
+## Properties
+
+- buildType
```

Comment:
Could you give some examples of what a `buildType` might be?

Reply:
A URI indicating what type of build was performed. It determines the meaning of how to interpret the inputs of the build - e.g. how to handle the entry point or how to carry out the build